internal/apischema/openai/openai.go

```diff
@@ -1580,6 +1582,9 @@ type GCPVertexAIGenerationConfig struct {
 	//
 	// https://cloud.google.com/vertex-ai/docs/reference/rest/v1/GenerationConfig#ThinkingConfig
 	ThinkingConfig *genai.ThinkingConfig `json:"thinkingConfig,omitzero"`
+
+	// Detail here is to set global media resolution in gemini models: https://ai.google.dev/api/caching#MediaResolution
+	Detail genai.MediaResolution `json:"detail,omitempty"`
```

Comment:
`details` can be confusing without context at the top level, since it is in the GCP vendor field section we should keep the name `medium_resolution`.